account_due_list/migrations/8.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# © 2016 Savoir-faire Linux
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+column_drops = {
+    ('account_move_line', 'day'),
+}
+
+
+@openupgrade.migrate()
+def migrate(cr, version):
+    if not version:
+        return
+
+    openupgrade.drop_columns(cr, column_drops)
```

Comment:
@pedrobaeza this is absolutely another context. Just look a few lines above of the code you pointed. What it does is drop the column when the field changes from stored to not stored.

In my case, the field does not exist anymore.